### ISSUE TITLE
Version 0.4.3 is still using CRLF

### ISSUE THREAD

**Issue by meowtec** [TypeStrong/dts-bundle#35]
```shell
npm i dts-bundle -g
dts-bundle 
env: node\r: No such file or directory
```

I have to edit global node_modules/dts-bundle/lib/dts-bundle.js, replace all CRLF to LF, to use dts-bundle

**Comment by tolemac** [comment]
@meowtec thanks for report it.

I think that it is a linux issue, true?

I don't know why the tests passed in travis that runs on linux ...
I will try to fix it ASAP.

**Comment by meowtec** [comment]
Maybe your local files were still CRLF, but git auto transform CRLF to LF so that tests passed in travis?

**Comment by bjouhier** [comment]
I just installed it with `npm install -g dts-bundle` on OSX. After install:
```sh
$ dts-bundle --name my-project.d.ts --main out/index.d.ts
env: node\r: No such file or directory
```
I edited the `bin/dts-bundle` file and changed the line endings to Unix style. After that it worked:
```sh
dts-bundle --name my-project.d.ts --main out/index.d.ts
dts-bundle version 0.4.3
Export TypeScript .d.ts files as an external module definition
```

BTW, cool tool 😄. I just had to do a small fix in one of my files and I was able to generate a valid `.d.ts` for my project.

**Comment by tolemac** [comment]
Sorry for the delay ... I have to fix it.

**Comment by tolemac** [comment]
@meowtec @bjouhier I could see that its a problem building the npm package, the repository it's ok.
I have released 0.5.0 version of npm package, I have download it and it hasn't CRLF line endings.

Close the issue, feel free to reopen if you want.
